helper/network-server-helper.h

```diff
@@ -81,12 +91,11 @@ class NetworkServerHelper
 
     ObjectFactory m_factory;
 
-    NodeContainer m_gateways; //!< Set of gateways to connect to this NS
+    std::list<std::pair<Ptr<NetDevice>, Ptr<Node>>>
```

Comment:
Yes, I'd like to be able to support any type of connection (Csma for instance) as in reality GW-NS traffic is transported by UDP/TCP or even higher layers (MQTT). In that way the NS can just use the IP of each GW and one NetDevice on its side, instead of having to track which NetDevice to use as it happens now with P2P.